ompi/errhandler/errcode-internal.c

```diff
@@ -285,6 +288,15 @@ int ompi_errcode_intern_init (void)
     opal_pointer_array_set_item(&ompi_errcodes_intern, ompi_err_rma_flavor_intern.index,
                                 &ompi_err_rma_flavor_intern);
 
+    OBJ_CONSTRUCT(&ompi_err_rma_nb_pending_intern, ompi_errcode_intern_t);
+    ompi_err_rma_nb_pending_intern.code = OMPI_ERR_RMA_NB_PENDING;
+    ompi_err_rma_nb_pending_intern.mpi_code = MPI_ERR_RMA_NB_PENDING;
+    ompi_err_rma_nb_pending_intern.index = pos++;
+    opal_string_copy(ompi_err_rma_nb_pending_intern.errstring,
+                     "OMPI_ERR_RMA_NB_PENDING", OMPI_MAX_ERROR_STRING);
+    opal_pointer_array_set_item(&ompi_errcodes_intern, ompi_err_rma_nb_pending_intern.index,
+                                &ompi_err_rma_nb_pending_intern);
+
```

Comment:
Does `ompi_err_rma_fatal_intern` also need a block of code like this to initialise and OMPI object for it and add it to the `ompi_errcodes_intern` list?

If `ompi_err_rma_fatal_intern` is never used, then we should remove its declaration (line 63 of this diff).

Reply:
We are not using `ompi_err_rma_fatal_intern` at the moment. So it can be removed and added in the future if required.